lib/content-services/src/lib/search/components/search-text/search-text.component.scss

```diff
@@ -1,6 +1,5 @@
 .adf-search-text {
     .mat-form-field {
         width: 100%;
-        max-width: 168px;
```

Comment:
I wonder a bit about the bug that introduced this fix before your PR, maybe we should use `hidden` instead of `ngIf` for the clear button to ensure that layout won't "jump" in case this component is used in similar scenario as in that bug

Reply:
I checked in application, it's not jumping anymore

Reply:
But the bug was introduce in the HX app, so you would need to check it or apply suggested changes